### PR TITLE
upstream latest changes after version release

### PR DESCRIPTION
**Pull request by paullatzelsperger**
After releasing version 0.3.2 there where a couple of last minute fixes and changes that needed to be done, as well as the merge commits after the release.

These should now be upstreamed.

_the sheer amount of commits is because of the merge `main` -> `develop`_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +49,25 @@ corresponding [documentation](/docs/migration/Version_0.1.x_0.3.x.md).
...
+- cp-adapter : code review, rollbacke name change (#664)
+- Feature/cp adapter task 355 356 357 (#621)
+- Add Validity Mapping in ContractDefinitionStepDefs class
+- Add feature and create SendAnOfferwithoutConstraints method in class negotiationSteps
+- Add validity attribute in class ContractDefinition
+- Add Validity Mapping in ContractDefinitionStepDefs class
+- Add feature and create SendAnOfferwithoutConstraints method in class negotiationSteps
+- Add validity attribute in class ContractDefinition
+- Local TXDC Setup Documentation (#618)
+- Feature: Sftp Provisioner and Client (#554)
```
Comment (florianrusch-zf) [inline CHANGELOG.md:69]: I think there is something wrong. The points are not doubled. Have a look at the lines (new) 33-50.
Same for the Changed and Fixed sections

File: NOTICE.md [hunk 2]
```diff
@@ -26,7 +26,6 @@ SPDX-License-Identifier: Apache-2.0
 
 ## Source Code
 
-The project maintains the following source code repositories
```
Comment (florianrusch-zf) [inline NOTICE.md:29]: I wouldn't remove this :D

File: edc-tests/cucumber/build.gradle.kts [hunk 3]
```diff
@@ -34,3 +34,8 @@ tasks.withType(Test::class) {
         System.getProperty("cucumber") == "true"
     }
 }
+
+// do not publish
+edcBuild {
+    publish.set(false)
+}
```
Comment (florianrusch-zf) [inline edc-tests/cucumber/build.gradle.kts:41]: Missing empty line at EOF

File: edc-tests/e2e-tests/build.gradle.kts [hunk 4]
```diff
@@ -30,3 +30,8 @@ dependencies {
     testImplementation(edc.api.catalog)
     testImplementation(testFixtures(edc.junit))
 }
+
+// do not publish
+edcBuild {
+    publish.set(false)
+}
```
Comment (florianrusch-zf) [inline edc-tests/e2e-tests/build.gradle.kts:37]: Missing empty line at EOF

File: edc-tests/runtime/build.gradle.kts [hunk 5]
```diff
@@ -37,3 +37,8 @@ tasks.withType<com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar> {
     mergeServiceFiles()
     archiveFileName.set("app.jar")
 }
+
+// do not publish
+edcBuild {
+    publish.set(false)
+}
```
Comment (florianrusch-zf) [inline edc-tests/runtime/build.gradle.kts:44]: Missing empty line at EOF

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

**Comment by florianrusch-zf** [comment]
@SebastianBezold you merged it to early...there is at least one wrong change in the files.

File: CHANGELOG.md [hunk 6]
```diff
@@ -72,6 +78,11 @@ corresponding [documentation](/docs/migration/Version_0.1.x_0.3.x.md).
 - Fix not working docu link in README.md
 - Fix typo in control-plane adapter README
 
+- bugfix: Fix slow AES encryption (#746)
+- Fix typo in tractusx-connector values.yaml comment
+- Fix not working docu link in README.md
+- Fix typo in control-plane adapter README
```
Comment (florianrusch-zf) [inline CHANGELOG.md:84]: Still a doublet

**Review by florianrusch-zf — COMMENTED** [review]
@paullatzelsperger will you fix this in another PR?

**Comment by SebastianBezold** [comment]
> @SebastianBezold you merged it to early...there is at least one wrong change in the files.

Oh I'm sorry @florianrusch-zf , should we revert?

**Pull request merged**